content/xr-dev/virtual-reality/zero-to-quest.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Zero To Quest - Meta XR SDK
+---
+
+# Zero to Quest Meta XR SDK
+
+## Overview
+Developing for virtual reality can be a daunting task, especially for beginners. This guide aims to simplify the process by walking you through the necessary steps to develop VR applications for Meta's Quest devices, using the [Meta Building Blocks SDK](https://assetstore.unity.com/packages/tools/integration/meta-xr-all-in-one-sdk-269657?srsltid=AfmBOopGcg_wSYbVGG4iWQu9dGdDVLT57kF8zgaTrSOuXcQgZE1cRQFo).
+
+## Setting Up Your Unity Project
+- **Create a New Unity Project:**
+  - Choose either Unity's Universal Render Pipeline (URP) or the Built-in Render Pipeline.
+  - Ensure that the Android Module is installed:
+    - Open Unity Hub and click on "Installs."
+    - Select the settings icon next to the Unity version you plan to use.
+    - Click on "Add Modules" and install "Android Build Support."
+    - If the Android module is already installed, it will be visible at the bottom of the Unity version details.
+
+![Android Build Setting](/images/unity/Zero-to-quest-meta-sdk/Add-Modules.png)
+
+- **Switch Platform to Android:**
+  - In your Unity project, navigate to `File > Build Settings`.
+  - Select "Android" and click "Switch Platform."
+
+## Installing the Meta XR SDK
+- **Download and Install the Meta XR All-in-One SDK:**
+  - Follow this [link](https://assetstore.unity.com/packages/tools/integration/meta-xr-all-in-one-sdk-269657?srsltid=AfmBOopGcg_wSYbVGG4iWQu9dGdDVLT57kF8zgaTrSOuXcQgZE1cRQFo) to access the Meta XR All-in-One SDK on the Unity Asset Store.
```

Comment:
In general, we want to avoid "this link", and prefer having the link be the name of the thing you are going to. Sort of a way of writing for the web instead of how we would speak out loud. So just something like "Get the Meta XR All-In-One SDK (that part links) from the Unity Asset Store"